projects/frontend-ui/shared-components/projects/gui/README.md

```diff
@@ -0,0 +1,69 @@
+The Shared Components contains:
+
+* **[Shared Components Library](projects/shared/README.md)** Library that holds UI Components.
+* **[Documentation UI](projects/documentation-ui/src)** Reference UI Wrapper and Documentation Implementation
+
+
+## Running and Testing
+
+You can use the reference implementation in `projects/documentation-ui` as showcase of the library.
```

Comment:
not sure yet about documentation-ui, my current understanding is we go without documentation-ui, so lets remove it for now

Reply:
removed all references to documentation UI.

Reply:
there is still documentation-ui dir structure

Reply:
there is still documentation-ui dir structure

Reply:
in what file do you see it referenced?